Healthy/Classes/Utilities/ValidatorProtocol/HealthyValidationRule .swift

```diff
@@ -0,0 +1,80 @@
+//
+//  HealthyValidationRule .swift
+//  Healthy
+//
+
+//
+
+import Foundation
+public enum HealthyValidationRule {
```

Comment:
Whether this is good, but it violates Open-Closed Principle. This will get much bigger when more cases are added. 

Consider something like this
```swift
protocol ValidationRule {
    typealias ValueType = String

    func validate(_ value: ValueType) throws
}

struct RegexValidationRule: ValidationRule {
    let field: String
    let regex: String

    func validate(_ value: String) throws {
        guard value.range(of: regex,
                          options: .regularExpression,
                          range: nil,
                          locale: nil) != nil else {
            throw ValidationError(errorDescription: "The \(field) is not valid")
        }
    }
}

struct NumbericValidationRule: ValidationRule {
    let field: String

    func validate(_ value: ValueType) throws {
        if value.rangeOfCharacter(from: CharacterSet.decimalDigits.inverted) != nil {
            throw ValidationError(errorDescription: "\(field) should contain numbers only")
        }
    }
}
```